elasticdl/python/common/model_utils.py

```diff
@@ -191,6 +191,31 @@ def get_model_spec(
     )
 
 
+def get_training_func_spec(
+    model_zoo, model_def, feed, custom_data_reader,
```

Comment:
Please add more description about the parameters.
Is `model_zoo` a string type?
Is `model_def` a piece of code or a model class name.
Is `feed` a method name in the model definition file?

Reply:
Yeah, I have added descriptions for those arguments.